language/Portuguese_Brazil/strings.po

```diff
@@ -906,11 +906,11 @@ msgstr "WAN\tImperfeita\n"
 
 #: src/frontend/mame/ui/selgame.cpp:964
 msgid "Timing\tUnimplemented\n"
-msgstr ""
+msgstr "Temporizador\tNão Implementado\n"
```

Comment:
The correct translation here is "Temporazação" meaning "Timing" instead of the above which means "Timer"